diff --git a/src/vulkan/gpu.c b/src/vulkan/gpu.c
index 57982199..0d0bebb0 100644
--- a/src/vulkan/gpu.c
+++ b/src/vulkan/gpu.c
@@ -386,13 +386,6 @@ static const VkFilter filters[PL_TEX_SAMPLE_MODE_COUNT] = {
 
 static inline struct pl_spirv_version get_spirv_version(const struct vk_ctx *vk)
 {
-    if (vk->api_ver >= VK_API_VERSION_1_4) {
-        return (struct pl_spirv_version) {
-            .env_version = VK_API_VERSION_1_4,
-            .spv_version = PL_SPV_VERSION(1, 6),
-        };
-    }
-
     if (vk->api_ver >= VK_API_VERSION_1_3) {
         const VkPhysicalDeviceMaintenance4Features *device_maintenance4;
         device_maintenance4 = vk_find_struct(&vk->features,
